scripts/helpers.js

```diff
@@ -2,124 +2,119 @@
-      + '<i class="fa fa-chevron-left"></i><span>' + this.__('page.prev') + '</span></a>';
+    result += `<a href="${keys[index - 1]}" class="article-footer-prev" title="${this.__(prefix + list[keys[index - 1]])}"><i class="fa fa-chevron-left"></i><span>${this.__('page.prev')}</span></a>`;
   }
 
   if (index < keys.length - 1) {
     result += '<a href="' + keys[index + 1] + '" class="article-footer-next" title="' + this.__(prefix + list[keys[index + 1]]) + '">'
       + '<span>' + this.__('page.next') + '</span><i class="fa fa-chevron-right"></i></a>';
+    result += `<a href="${keys[index + 1]}" class="article-footer-next" title="${this.__(prefix + list[keys[index + 1]])}"><span>${this.__('page.next')}</span><i class="fa fa-chevron-right"></i></a>`;
   }
 
   return result;
 });
 
 hexo.extend.helper.register('doc_sidebar', function(className) {
-  var type = this.page.canonical_path.split('/')[0];
-  var sidebar = this.site.data.sidebar[type];
-  var path = pathFn.basename(this.path);
-  var result = '';
-  var self = this;
-  var prefix = 'sidebar.' + type + '.';
-
-  _.each(sidebar, function(menu, title) {
-    result += '<strong class="' + className + '-title">' + self.__(prefix + title) + '</strong>';
-
-    _.each(menu, function(link, text) {
-      var itemClass = className + '-link';
+  const type = this.page.canonical_path.split('/')[0];
+  const sidebar = this.site.data.sidebar[type];
+  const path = pathFn.basename(this.path);
+  let result = '';
+  const self = this;
+  const prefix = 'sidebar.' + type + '.';
+
+  _.each(sidebar, (menu, title) => {
+    result += `<strong class="${className}-title">${self.__(prefix + title)}</strong>`;
+
+    _.each(menu, (link, text) => {
+      let itemClass = className + '-link';
       if (link === path) itemClass += ' current';
 
-      result += '<a href="' + link + '" class="' + itemClass + '">' + self.__(prefix + text) + '</a>';
+      result += `<a href="${link}" class="${itemClass}">${self.__(prefix + text)}</a>`;
     });
   });
 
   return result;
 });
 
 hexo.extend.helper.register('header_menu', function(className) {
-  var menu = this.site.data.menu;
-  var result = '';
-  var self = this;
-  var lang = this.page.lang;
-  var isEnglish = lang === 'en';
+  const menu = this.site.data.menu;
+  let result = '';
+  const self = this;
+  const lang = this.page.lang;
+  const isEnglish = lang === 'en';
 
-  _.each(menu, function(path, title) {
+  _.each(menu, (path, title) => {
     if (!isEnglish && ~localizedPath.indexOf(title)) path = lang + path;
 
-    result += '<a href="' + self.url_for(path) + '" class="' + className + '-link">' + self.__('menu.' + title) + '</a>';
+    result += `<a href="${self.url_for(path)}" class="${className}-link">${self.__('menu.' + title)}</a>`;
   });
 
   return result;
 });
 
+// Will be replace with full_url_for after hexo v4 release
```

Comment:
this repo is using master branch of hexo, so full_url_for should be supported?

Reply:
Added:

```js
const full_url_for = hexo.extend.helper.get('full_url_for').bind(hexo);
```

It is not recommends to use `hexo-util` since it is extraneous.